src/components/organisms/Header/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export const Header = (props) => {
+  return (
+    <header>
+      <Link to="/">Home</Link>
+      <Link to={`/site/profile/${props.id}`}>Profile</Link>
```

Comment:
Header doesn't need props,
In this case, edit profile or profile link, would redirect to the logged in user profile, so we would be getting that directly in here, when this component mounts, so we don't need any props here

Just export the someId (should be profileId) and use it here directly

Reply:
Done